pages/common/git-commit.md

```diff
@@ -7,6 +7,10 @@
 
 `git commit -m "{{message}}"`
 
+- Commit staged files with a message read from file:
+
+`git commit --file {{path/to/my/file}}`
```

Comment:
I think we should be more explicit in this file content to avoid confusion with the last example:
```suggestion
`git commit --file {{path/to/commit_message_file}}`
```